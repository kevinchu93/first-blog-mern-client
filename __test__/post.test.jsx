import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Post from '../src/app/posts/post';


describe('Post - shallow', () => {
  const mockPost = {
    title: 'title',
    body: 'body',
    author: 'author',
  };
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Post post={mockPost} />);
  });
  test('Post component should render as expected - snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});

