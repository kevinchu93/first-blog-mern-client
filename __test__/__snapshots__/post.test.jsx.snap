// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post - shallow Post component should render as expected - snapshot 1`] = `
<li>
  <h2>
    <Link
      className="title"
      replace={false}
      to="/posts/post_id"
    >
      title
    </Link>
  </h2>
  <p>
    body
  </p>
  <p
    className="author"
  >
    author
  </p>
</li>
`;
